embassy-stm32/src/flash/mod.rs

```diff
@@ -1,15 +1,24 @@
 use embedded_storage::nor_flash::{NorFlashError, NorFlashErrorKind};
 
+#[cfg(all(feature = "nightly", flash_f4))]
```

Comment:
async support shouldn't be gated to nightly. 

Only the async trait impls (for `embedded-storage-async` ) need nightly, Inherent methods don't need it.

Reply:
Year, the problem is that there are some of the chips that do not have the FLASH interrupt. Is there some cfg() that can determine if that is the case, or do I need to go through all the variants and determine which ones it is?

Reply:
gating to `flash_f4` is fine, I was referring to the nightly gate only.

Reply:
Of cause!